### ISSUE TITLE
damaged mechs will not deploy in battle

### ISSUE THREAD

**Issue by bczretribution** [MegaMek/mekhq#655]
### Environment
mekhq 43.7 rc1

### Description
In the attached file, night hawk#3 has 2 damaged double heat sinks.  It is part of a lance being deployed on an extraction mission.  Click start game, megamek will launch and load units, the night hawk in question will be shown among those deploying.  Hit done and start deployment, the night hawk in question is not shown down the right side and is not part of the initiative list,  it is nowhere to be found.

### Files
[glitch (2).zip](https://github.com/MegaMek/mekhq/files/1738323/glitch.2.zip)

**Comment by neoancient** [comment]
Custom unit: Hellcat II HCT-214P

**Comment by bczretribution** [comment]
crap sorry here it is.  not sure why that custom out of all of them is needed when the others arnt.
[glitch.zip](https://github.com/MegaMek/mekhq/files/1741990/glitch.zip)

**Comment by Qwertronix** [comment]
Does this issue persist if you save the game in MekHQ before starting the scenario, close MHQ, reload, and launch the scenario again?

**Comment by bczretribution** [comment]
yes, ive done that multiple times over the last few days to no effect.

**Comment by NickAragua** [comment]
Unfortunately, when I try to load up the scenario with the damaged mech, it deploys just fine. Did anything happen to it other than damaged heat sinks?

**Comment by bczretribution** [comment]
I don't fully remember but I think it was just that and some armor, might have lost an arm.  That's annoying, as I've mentioned in the slack its failed to deploy on multiple missions now.  at the time i made this report I thought it was the unfixed sinks that were the cause but now that the unit is fully repaired and still having the issue (and im pretty sure that pilot has never ejected) that might have been a misdiagnosis of the cause
